attributes/default.rb

```diff
@@ -23,3 +23,42 @@
 default['nexus_repository_manager']['nexus']['data']['etc']['path'] = default['nexus_repository_manager']['nexus']['data']['path'] + '/etc'
 default['nexus_repository_manager']['nexus']['data']['log']['path'] = default['nexus_repository_manager']['nexus']['data']['path'] + '/log'
 default['nexus_repository_manager']['nexus']['data']['tmp']['path'] = default['nexus_repository_manager']['nexus']['data']['path'] + '/tmp'
+
+# hazelcast configuration
```

Comment:
I'm not sure this should all go in the default configuration? What about people who want to deploy single nodes? Who don't want to deploy to AWS? Who don't have professional?

Reply:
Broke the default attributes into more targeted files. Still looking into refinements. (fc306c2)